command/config/config.go

```diff
@@ -326,16 +326,17 @@ type EnvConfig struct {
 	}
 
 	Settings struct {
-		DefaultDriver        string `envconfig:"DRONE_DEFAULT_DRIVER" default:"amazon"`
-		ReusePool            bool   `envconfig:"DRONE_REUSE_POOL" default:"false"`
-		BusyMaxAge           int64  `envconfig:"DRONE_SETTINGS_BUSY_MAX_AGE" default:"24"`
-		FreeMaxAge           int64  `envconfig:"DRONE_SETTINGS_FREE_MAX_AGE" default:"720"`
-		MinPoolSize          int    `envconfig:"DRONE_MIN_POOL_SIZE" default:"1"`
-		MaxPoolSize          int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
-		EnableAutoPool       bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
-		HarnessTestBinaryURI string `envconfig:"DRONE_HARNESS_TEST_BINARY_URI"`
-		PluginBinaryURI      string `envconfig:"DRONE_PLUGIN_BINARY_URI" default:"https://github.com/drone/plugin/releases/download/v0.3.6-beta"`
-		PurgerTime           int64  `envconfig:"DRONE_PURGER_TIME_MINUTES" default:"30"`
+		DefaultDriver          string `envconfig:"DRONE_DEFAULT_DRIVER" default:"amazon"`
+		ReusePool              bool   `envconfig:"DRONE_REUSE_POOL" default:"false"`
+		BusyMaxAge             int64  `envconfig:"DRONE_SETTINGS_BUSY_MAX_AGE" default:"24"`
+		FreeMaxAge             int64  `envconfig:"DRONE_SETTINGS_FREE_MAX_AGE" default:"720"`
+		MinPoolSize            int    `envconfig:"DRONE_MIN_POOL_SIZE" default:"1"`
+		MaxPoolSize            int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
+		EnableAutoPool         bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
+		HarnessTestBinaryURI   string `envconfig:"DRONE_HARNESS_TEST_BINARY_URI"`
```

Comment:
Check where is this URI being populated from. You might not be required to use a default value for auto-injection.

Reply:
I couldnt get where this is populating( I saw ng prod manifests too, couldn't find this env var)
but in this drone-runner-aws repo they are populating the value if this env is empty , so instead of populating their i kept it as default value.
![Uploading Screenshot 2024-08-19 at 12.27.47 AM.png…]()